### PR TITLE
#161622597 Loaded images not displaying bug fix

### PR DESCRIPTION
**Pull request by mindsworth**
#### What does this PR do?
This PR is used to fix the bug of loaded images not displaying.

#### Description of Task to be completed?
- installed file-loader
- installed @babel/node
- removed devServer config from webpack.config.dev file
- changed devTool config to eval-source-map in webpack.config.dev file
- installed path and re-configured the paths webpack config files

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app will open on port ```http://localhost:8080``` on your browser
- From the old or a new component, load the image using `````` tag.

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161622597](https://www.pivotaltracker.com/story/show/161622597)

#### Screenshots (if appropriate)
- None

#### Questions:
- None

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
-// react libraries
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider} from 'react-redux';
-import configureStore from '../src/store/configureStore'
 
-// components
-import Router from './components/Router';
+// IMPORT STYLESHEET
```
Comment (KaiserPhemi) [inline src/index.js]: #### Critical
Please remove unneeded lines of code instead of commenting them out.

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
 
-ReactDOM.render((
-    <Provider store={store}>
-    <Router />
-    </Provider>),
-   document.getElementById('root'));
+ReactDOM.render(
+  <AppRouter />,
+  document.getElementById('root')
+);
```
Comment (KaiserPhemi) [inline src/index.js]: #### Nitpick
Add a new line at the end of the file.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (mindsworth) [reply src/index.js]: fixed

File: src/index.js (hunk 2 above)
Comment (mindsworth) [reply src/index.js]: fixed

**Review by andela-phemmy — APPROVED** [review]
Looking Good To Me

**Review by koiic — APPROVED** [review]
Looking Good To Me

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**